### PR TITLE
Issue #383: updated minishift install guide per feedback. resolves #383

### LINKED ISSUE #383: Update minishift install instructions with template callouts
* Provide a better reference to template instead of just saying `here`.
* Consider making a better call out to clearing key cloak fields.

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

File: docs/topics/minishift-install-create-launchpad-app.adoc [hunk 1]
```diff
@@ -27,14 +27,14 @@ image::minishift_processtemplate.png[Process Template]
...
 +
 WARNING: You must clear these fields out for the {launcher} application on your {OpenShiftLocal} to be configured correctly.
 
 ** _Catalog Git Repository_ and _Catalog Git Reference_ should be generally left as-is, unless you are developing against a specific fork you would like to use.
-** _{launcher} Frontend Host_ must be updated to replace `$PROJECTNAME` with the name of your project. You also need to update `$HOSTNAMEORIP` in the string to the IP address of your OpenShift Console URL. This example uses `+++192.168.42.152+++` and `{launcher}`.
-+
-WARNING: Before proceeding to the next steps, confirm all the fields are correct.
+** _{launcher} Frontend Host_ must be entered. Usually this takes the form of `launchpad-frontend-$PROJECTNAME.$HOSTNAMEORIP.nip.io`, replacing `$PROJECTNAME` with the name of your project and `$HOSTNAMEORIP` with the IP address of your OpenShift Console URL. This example uses `my-launcher` and `+++192.168.42.152+++` to make `launchpad-frontend-my-launcher.192.168.42.152.nip.io`.
+
+. Before proceeding to the next steps, confirm all the fields are correct. Also confirm that _KeyCloak URL_ and _KeyCloak Realm_ **have been be cleared out**.
```
Comment (gastaldi) [inline docs/topics/minishift-install-create-launchpad-app.adoc]: *have been cleared out

**Comment by rhoads-zach** [comment]
thanks @gastaldi .

Fixed the issue.

**Comment by inoxx03** [comment]
LGTM, merging...

**Comment by inoxx03** [comment]
merged: cbf553e7db57d580048969478475d38084c547c5

**Pull request closed without merging**